digdag-storage-gcs/src/main/java/io/digdag/storage/gcs/GCSStorage.java

```diff
@@ -165,6 +165,18 @@ public void list(String objectPrefix, FileListing callback)
         return Optional.of(DirectUploadHandle.of(url));
     }
 
+    @Override
+    public Long getDirectDownloadExpiration()
+    {
+        return config.get("direct_download_expiration", Long.class, 10L*60);
```

Comment:
Could you explain about default value `10L*60`?

Reply:
https://github.com/treasure-data/digdag/blob/432517d0141959a7bc906790ce99233bc39544ce/digdag-storage-gcs/src/main/java/io/digdag/storage/gcs/GCSStorage.java#L159
@yoyama  Because it is just original implementation use `10L*60` as default value